src/modules/form/util/formUtil.ts

```diff
@@ -1476,6 +1476,36 @@ export const parseOccurrencePointFormDefinition = (
 
   return { displayTitle, isEditable, readOnlyDefinition };
 };
+
+export const getLeftNavItems = (
```

Comment:
Nice refactor. I think we can call it something more descriptive though, like `getFormStepperItems`? The component itself is called `FormStepper`. Usually we use "nav" to talk about the left nav on dashboards, so it could be confusing